recon_surf/rewrite_mc_surface.py

```diff
@@ -54,19 +53,19 @@ def options_parse():
 
 
 def resafe_surface(insurf: str, outsurf: str, pretess: str) -> None:
-    """Take path to insurf and rewrite it to outsurf thereby fixing vertex locs flag error.
+    """
+    Take path to insurf and rewrite it to outsurf thereby fixing vertex locs flag error.
 
-    (scannerRAS instead of surfaceRAS after marching cube)
+    (scannerRAS instead of surfaceRAS after marching cube).
```

Comment:
@m-reuter this line makes no sense to me. maybe you can propose how to fix this line.

Reply:
I have no idea, this was implemented 4 years ago by @LeHenschel  here 59f3959b1d3961334